source-mongodb/discovery.go

```diff
@@ -24,10 +24,12 @@ const idProperty = "_id"
 const (
 	metaProperty    = "_meta"
 	deletedProperty = "deleted"
+	opProperty      = "op"
 )
 
 type documentMetadata struct {
-	Deleted bool `json:"deleted,omitempty" jsonschema:"title=Delete Flag,description=True if the document has been deleted, unset otherwise."`
+	Deleted bool `json:"deleted,omitempty" jsonschema:"title=Delete Flag (deprecated, use op instead),description=True if the document has been deleted, unset otherwise."`
+	Op bool `json:"op,omitempty" jsonschema:"title=Change Operation,description=Change operation type: 'c' Create/Insert, 'u' Update, 'd' Delete."`
```

Comment:
It would be good to include this `_meta/op` property in the minimal discovered schema, since it is controlled by the connector.

Reply:
@williamhbaker this is the line that does that, the minimal schema of mongodb is generated from this struct's reflection

Reply:
Ah I didn't even notice that! Thanks.